.rspec

```diff
@@ -1 +1,2 @@
 --require spec_helper
+--format NyanCatFormatter
```

Comment:
This makes a beautiful kitty rainbow when we run RSpec. Can delete if anyone hates it!

Reply:
![](https://media3.giphy.com/media/sIIhZliB2McAo/giphy.gif?cid=5a38a5a2c3e31c73369a3d8f6cfe563e67fe2b18b4fe73e1&rid=giphy.gif)